### PR TITLE
Remove repo form collections

### PR DESCRIPTION
**Pull request by ymh6315431**

### CODE REVIEW

File: app/javascript/components/collections/CollectionsRepoList.vue [hunk 1]
```diff
@@ -119,13 +127,21 @@
 </template>
 <script setup>
   import { computed, ref, onMounted, inject } from 'vue'
-
   import RepoItem from '../shared/RepoItem.vue'
   import ApplicationSpaceItem from '../application_spaces/ApplicationSpaceItem.vue'
+  import jwtFetch from '../../packs/jwtFetch'
```
Comment (hiveer) [inline app/javascript/components/collections/CollectionsRepoList.vue:132]: jwtFetch has been refactored, please merge latest code and update it accordingly

File: app/javascript/components/collections/CollectionsRepoList.vue [hunk 2]
```diff
@@ -136,6 +152,40 @@
...
+    }
+  }
+
+  const removeRepoFromCollections = async (id) => {
+    const removeRepoData = {
+      repo_ids: [id]
+    }
+    const options = { method: "DELETE", body: JSON.stringify(removeRepoData) }
+    const url = `${csghubServer}/api/v1/collections/${props.collectionsId}/repos`
+    const response = await jwtFetch(url, options)
```
Comment (hiveer) [inline app/javascript/components/collections/CollectionsRepoList.vue:179]: 同上

File: app/javascript/components/collections/CollectionsRepoList.vue (hunk 1 above)
Comment (ymh6315431) [reply app/javascript/components/collections/CollectionsRepoList.vue]: down

File: app/javascript/components/collections/CollectionsRepoList.vue (hunk 2 above)
Comment (ymh6315431) [reply app/javascript/components/collections/CollectionsRepoList.vue]: down

File: app/javascript/components/collections/CollectionsDetail.vue [hunk 3]
```diff
@@ -78,6 +78,8 @@
         <CollectionsRepoList
           v-if="showRepoList"
           :repositories="collectionData.repositories"
+          :collectionsId="collectionsId"
+          :settingsVisibility="settingsVisibility"
```
Comment (hiveer) [inline app/javascript/components/collections/CollectionsDetail.vue:82]: Please update the name 'settingsVisibility' to  'canManage'

Comment (ymh6315431) [reply app/javascript/components/collections/CollectionsDetail.vue]: done

**Review by hiveer — APPROVED** [review]
LGTM

**Pull request merged**